src/main/scala/latis/service/hapi/AboutService.scala

```diff
@@ -0,0 +1,31 @@
+package latis.service.hapi
+
+import cats.effect.Concurrent
+import io.circe.syntax._
+import org.http4s.HttpRoutes
+import org.http4s.circe._
+import org.http4s.dsl.Http4sDsl
+
+import latis.util.LatisConfig
+import latis.util.LatisException
+
+/** Implements the `/about` endpoint. */
+class AboutService[F[_]: Concurrent] extends Http4sDsl[F] {
+
+  val service: HttpRoutes[F] =
+    HttpRoutes.of[F] {
+      case GET -> Root / "about" =>
+        Ok(
+          AboutResponse(
+            HapiService.version,
+            Status.`1200`,
+            LatisConfig.get("latis.about.id").getOrElse(throw LatisException("No 'id' configured")),
+            LatisConfig.get("latis.about.title").getOrElse(throw LatisException("No 'title' configured")),
+            LatisConfig.get("latis.about.contact").getOrElse(throw LatisException("No 'contact' configured")),
+            LatisConfig.get("latis.about.description"),
+            LatisConfig.get("latis.about.contactId"),
+            LatisConfig.get("latis.about.citation")
```

Comment:
Any thoughts on these config paths? I went with `latis.about.*` because every other config was already under `latis` and `about` seemed natural enough. I think it's fine.

Reply:
I'd go with `latis.hapi.about` so we can reserve the `latis.about` namespace for our own purposes.

Reply:
Cool, I like that.